### PR TITLE
add group balance command

### PR DESCRIPTION
**Pull request by mp-singh**
what the commit says

### CODE REVIEW

File: bin/bot/src/discord/slash/bank.rs [hunk 1]
```diff
@@ -73,13 +80,41 @@ pub async fn run(ctx: &Context, command: &ApplicationCommandInteraction) -> sere
...
+            member: UserId::from(0),
+        }
+    )
+    .await else {
+        return interaction.reply("Failed to fetch balance").await;
+    };
+
+    interaction
+        .reply(format!(
+            "The Group has:\n```Cash:      {}\n```",
```
Comment (mp-singh) [inline bin/bot/src/discord/slash/bank.rs:112]: probably dont all the extra white space between cash and the value?

File: bin/bot/src/discord/slash/bank.rs [hunk 2]
```diff
@@ -33,6 +34,12 @@ pub fn register(command: &mut CreateApplicationCommand) -> &mut CreateApplicatio
                         .required(true)
                 })
                 .allow_public()
+        }).create_option(|option| {
+            option
+            .name("group_balance")
+            .description("View the Group balance")
```
Comment (mp-singh) [inline bin/bot/src/discord/slash/bank.rs:40]: group lowercase?

Comment (BrettMayson) [reply bin/bot/src/discord/slash/bank.rs]: imo should do brodsky as the member to see the group fund

Comment (mp-singh) [reply bin/bot/src/discord/slash/bank.rs]: i just did!

**Review by BrettMayson — APPROVED** [review]

**Pull request merged**